src/facade/dragonfly_connection.cc

```diff
@@ -1176,11 +1180,19 @@ void Connection::Migrate(util::fb2::ProactorBase* dest) {
   // connections
   CHECK(!cc_->async_dispatch);
   CHECK_EQ(cc_->subscriptions, 0);    // are bound to thread local caches
+  CHECK_EQ(self_.use_count(), 1u);    // references cache our thread and backpressure
```

Comment:
Just making sure: this must be 1 because there are no subscriptions, right?

Reply:
Yes, `CHECK_EQ(cc_->subscriptions, 0);` is actually the stronger condition here